projects/cloud/src/app/model/authorization_config.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 Ping Identity
+//
```

Comment:
Are these copyrights correct?

Reply:
yes - we should add them to license.

Reply:
I have now slightly modified authorization service but that is OK under MIT license.